Boot2Docker Status/AppDelegate.m

```diff
@@ -30,6 +31,12 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
     
     _standardUserDefaults = [NSUserDefaults standardUserDefaults];
     
+	_darkMode = NO;
+    NSString *osxMode = [_standardUserDefaults stringForKey:@"AppleInterfaceStyle"];
```

Comment:
Please put this into the `renderIcon` method as well, so the icon will respond to changes from light mode to dark mode and vice versa (likely only affects flux users).